testing/src/main/java/io/grpc/testing/TestMethodDescriptors.java

```diff
@@ -40,20 +40,29 @@ private TestMethodDescriptors() {}
    */
   @ExperimentalApi("https://github.com/grpc/grpc-java/issues/2600")
   public static MethodDescriptor<Void, Void> voidMethod() {
-    return TestMethodDescriptors.<Void, Void>noopMethod();
+    return MethodDescriptor.<Void, Void>newBuilder()
+        .setType(MethodType.UNARY)
+        .setFullMethodName(MethodDescriptor.generateFullMethodName("service_foo", "method_bar"))
+        .setRequestMarshaller(TestMethodDescriptors.voidMarshaller())
+        .setResponseMarshaller(TestMethodDescriptors.voidMarshaller())
+        .build();
   }
 
   /**
    * Creates a new method descriptor that always creates zero length messages, and always parses to
    * null objects.
    *
+   * @deprecated Use {@link #voidMethod()} instead or build one with {@link
```

Comment:
I thought we wanted them to use a descriptor from SimpleService if voidMethod didn't suffice.

Reply:
Right, we wanted them
1. if they don't care about message, use `voidMethod`
2. if they care about message, but don't care about its type, use the one from `SimpleService`
3. If they care about message and also its type, they should be able to build one for their own message type.

Will update the comment a little bit.

Reply:
And instead of building their own descriptor, I think we thought to suggest they just make a custom .proto for testing. Although it's up to them whatever they find easiest.